pkg/pool-weighted/contracts/managed/ManagedPool.sol

```diff
@@ -1057,7 +1062,7 @@ contract ManagedPool is BaseWeightedPool, ProtocolFeeCache, ReentrancyGuard, ICo
                 _tokenState[token],
                 startWeights[i],
                 endWeight,
-                denormWeightSum
+                FixedPoint.ONE
```

Comment:
Hm, interesting. This does seem correct. It'd be good to review the existing tests to get a sense of what the current coverage of all of this looks like.